src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -23,27 +24,30 @@ internal abstract partial class AbstractFindUsagesService
         /// Forwards <see cref="IStreamingFindLiteralReferencesProgress"/> calls to an
         /// <see cref="IFindUsagesContext"/> instance.
         /// </summary>
-        private class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgress
+        private sealed class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgress
         {
             private readonly IFindUsagesContext _context;
             private readonly DefinitionItem _definition;
-            private readonly ClassificationOptions _classificationOptions;
 
             public IStreamingProgressTracker ProgressTracker
                 => _context.ProgressTracker;
 
             public FindLiteralsProgressAdapter(
-                IFindUsagesContext context, DefinitionItem definition, ClassificationOptions classificationOptions)
+                IFindUsagesContext context, DefinitionItem definition)
             {
                 _context = context;
                 _definition = definition;
-                _classificationOptions = classificationOptions;
             }
 
             public async ValueTask OnReferenceFoundAsync(Document document, TextSpan span, CancellationToken cancellationToken)
             {
+                // TODO:
+                // var options = await _context.GetOptionsAsync(document.Project.Language, cancellationToken).ConfigureAwait(false);
```

Comment:
shouldn't this be part of the PR?

Reply:
No, unless you want the PR to be 3x larger :)

Reply:
I can certainly do that if you prefer.